FabNESO/tasks.py

```diff
@@ -90,16 +91,47 @@ def neso_ensemble(
         solver: Which NESO solver to use.
         conditions_file_name: Name of conditions XML file in configuration directory.
         mesh_file_name: Name of mesh XML in configuration directory.
+        parameter_scans is a set of parameters to sweep over. A slash separated list of
+        lower bound, upper bound, and steps.
```

Comment:
```suggestion
        **parameter_scans: The set of parameters to sweep over. A slash separated list of
            lower bound, upper bound, and steps.
```
To conform with [Google style docstring convention](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) (will make sure things render nicely in HTML documentation once we address #16).